src/main/java/com/faforever/client/fx/SimpleChangeListener.java

```diff
@@ -0,0 +1,15 @@
+package com.faforever.client.fx;
+
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+
+@FunctionalInterface
```

Comment:
This is a funny hack of functional interfaces I haven't seen before

Reply:
Really? Do you mean overloading the main functional method?

Or just having multiple methods in a functional interface?